registry/zookeeper/service_discovery_test.go

```diff
@@ -1,281 +1,131 @@
-/*
```

Comment:
pls do not delete the license header. If you do so, this is not a apache program.